### PR TITLE
cstyle fixes in src/istgtcontrol.c

### PR DESCRIPTION
**Pull request by rishabhp99**

### CODE REVIEW

**Review by vishnuitta — COMMENTED** [review]
can you please get me the output of cmd `./cstyle.pl src/istgtcontrol.c`?

**Comment by rishabhp99** [comment]
rishabh@rishabh-Inspiron-7460:~/istgt$ ./cstyle.pl src/istgtcontrol.c
rishabh@rishabh-Inspiron-7460:~/istgt$

**Comment by rishabhp99** [comment]
<pre><font color="#8AE234"><b>rishabh@rishabh-Inspiron-7460</b></font>:<font color="#729FCF"><b>~/istgt</b></font>$ ./cstyle.pl src/istgtcontrol.c
<font color="#8AE234"><b>rishabh@rishabh-Inspiron-7460</b></font>:<font color="#729FCF"><b>~/istgt</b></font>$ </pre>

**Comment by payes** [comment]
@rishabhp99 The build is failing, can you please resolve problems while running make.

**Comment by rishabhp99** [comment]
> @rishabhp99 The build is failing, can you please resolve problems while running make.

I am working on it.

**Comment by rishabhp99** [comment]
> @rishabhp99 The build is failing, can you please resolve problems while running make.

Please have a look at it.

File: src/istgtcontrol.c [hunk 1]
```diff
@@ -158,25 +159,26 @@ typedef enum {
...
+//	#define ARGS_DELIM " \t\r\n"
+#define	ARGS_DELIM " \t"
 
 static int
 uctl_readline(UCTL_Ptr uctl)
 {
 	ssize_t total;
 
-	total = istgt_readline_socket(uctl->sock, uctl->recvbuf, uctl->recvbufsize,
+	total = istgt_readline_socket
```
Comment (vishnuitta) [inline src/istgtcontrol.c]: this doesn't look good. Just bring one parameter to next line, and next line should be given just 4 spaces extra, i.e., 4 spaces away under 'total'

File: src/istgtcontrol.c [hunk 2]
```diff
@@ -186,17 +188,19 @@ uctl_writeline(UCTL_Ptr uctl)
...
-		return UCTL_CMD_ERR;
+		return (UCTL_CMD_ERR);
 	}
-	return UCTL_CMD_OK;
+	return (UCTL_CMD_OK);
 }
 
-static int uctl_snprintf(UCTL_Ptr uctl, const char *format, ...) __attribute__((__format__(__printf__, 2, 3)));
+static int uctl_snprintf(UCTL_Ptr uctl, const char *format, ...)
+__attribute__((__format__(__printf__, 2, 3)));
```
Comment (vishnuitta) [inline src/istgtcontrol.c]: here, 4 spaces initially

File: src/istgtcontrol.c [hunk 3]
```diff
@@ -186,17 +188,19 @@ uctl_writeline(UCTL_Ptr uctl)
 	ssize_t expect;
 
 	expect = strlen(uctl->sendbuf);
-	total = istgt_writeline_socket(uctl->sock, uctl->sendbuf, uctl->timeout);
+	total = istgt_writeline_socket
```
Comment (vishnuitta) [inline src/istgtcontrol.c]: here, same as previous comment, i.e., code doesn't look good

File: src/istgtcontrol.c [hunk 4]
```diff
@@ -600,96 +605,95 @@ exec_refresh(UCTL_Ptr uctl)
...
-        }
-        return UCTL_CMD_OK;
+	const char *delim = ARGS_DELIM;
+	char *arg;
+	char *result;
+	int rc;
+
+	/* send command */
+	if (uctl->iqn == NULL || uctl->lun < 0) {
+			return (UCTL_CMD_ERR);
```
Comment (vishnuitta) [inline src/istgtcontrol.c]: why the extra tab from here?

File: src/istgtcontrol.c [hunk 5]
```diff
@@ -600,96 +605,95 @@ exec_refresh(UCTL_Ptr uctl)
...
+	char *arg;
+	char *result;
+	int rc;
+
+	/* send command */
+	if (uctl->iqn == NULL || uctl->lun < 0) {
+		return (UCTL_CMD_ERR);
+	}
+	uctl_snprintf(uctl, "PERSIST \"%s\" %d %d\n", uctl->iqn,
+	uctl->lun, uctl->persistopt);
```
Comment (vishnuitta) [inline src/istgtcontrol.c]: 4 more spaces required here after the 'tab'

File: src/istgtcontrol.c [hunk 6]
```diff
@@ -600,96 +605,95 @@ exec_refresh(UCTL_Ptr uctl)
...
+	rc = uctl_readline(uctl);
+	if (rc != UCTL_CMD_OK) {
+		return (rc);
+	}
+	arg = trim_string(uctl->recvbuf);
+	result = strsepq(&arg, delim);
+	strupr(result);
+	if (strcmp(result, "OK") != 0) {
+		if (is_err_req_auth(uctl, arg))
+		return (UCTL_CMD_REQAUTH);
```
Comment (vishnuitta) [inline src/istgtcontrol.c]: an extra tab is required here.

File: src/istgtcontrol.c (hunk 5 above)
Comment (rishabhp99) [reply src/istgtcontrol.c]: In line 673 ?

File: src/istgtcontrol.c (hunk 1 above)
Comment (rishabhp99) [reply src/istgtcontrol.c]:
```
total = istgt_readline_socket (uctl->sock, uctl->recvbuf, uctl->recvbufsize,uctl->recvtmp, uctl->recvtmpsize,&uctl->recvtmpidx, &uctl->recvtmpcnt,
	    uctl->timeout);
```
Like this ?

File: src/istgtcontrol.c (hunk 6 above)
Comment (rishabhp99) [reply src/istgtcontrol.c]: Done

File: src/istgtcontrol.c (hunk 1 above)
Comment (vishnuitta) [reply src/istgtcontrol.c]: no.. we need it like this:
```
total = istgt_readline_socket(uctl->sock, uctl->recvbuf, uctl->recvbufsize, uctl->recvtmp,
    uctl->recvtmpsize, &uctl->recvtmpidx, &uctl->recvtmpcnt, uctl->timeout);
```

File: src/istgtcontrol.c [hunk 7]
```diff
@@ -199,8 +205,8 @@ uctl_writeline(UCTL_Ptr uctl)
 	return (UCTL_CMD_OK);
 }
 
-static int uctl_snprintf(UCTL_Ptr uctl, const char *format, ...)
-__attribute__((__format__(__printf__, 2, 3)));
+    static int uctl_snprintf(UCTL_Ptr uctl, const char *format, ...)
```
Comment (vishnuitta) [inline src/istgtcontrol.c:200]: here like:
```
static int uctl_snprintf(UCTL_Ptr uctl, const char *format, ...)
    __attribute__((__format__(__printf__, 2, 3)));
```

File: src/istgtcontrol.c [hunk 8]
```diff
@@ -635,29 +641,29 @@ exec_sync(UCTL_Ptr uctl)
 
 	/* send command */
 	if (uctl->iqn == NULL || uctl->lun < 0) {
-			return (UCTL_CMD_ERR);
-		}
+		return (UCTL_CMD_ERR);
+	}
 		uctl_snprintf(uctl, "SYNC \"%s\" %d\n", uctl->iqn, uctl->lun);
 		rc = uctl_writeline(uctl);
```
Comment (vishnuitta) [inline src/istgtcontrol.c]: no proper indentation here

File: src/istgtcontrol.c [hunk 9]
```diff
@@ -635,29 +641,29 @@ exec_sync(UCTL_Ptr uctl)
...
-		if (rc != UCTL_CMD_OK) {
-			return (rc);
-		}
+	if (rc != UCTL_CMD_OK) {
+		return (rc);
+	}
 
 		/* receive result */
 		rc = uctl_readline(uctl);
```
Comment (vishnuitta) [inline src/istgtcontrol.c]: here also

File: src/istgtcontrol.c [hunk 10]
```diff
@@ -600,96 +603,95 @@ exec_refresh(UCTL_Ptr uctl)
...
+	const char *delim = ARGS_DELIM;
+	char *arg;
+	char *result;
+	int rc;
+
+	/* send command */
+	if (uctl->iqn == NULL || uctl->lun < 0) {
+	    return (UCTL_CMD_ERR);
+	}
+	    uctl_snprintf(uctl, "PERSIST \"%s\" %d %d\n", uctl->iqn,
```
Comment (vishnuitta) [inline src/istgtcontrol.c]: indentation gone wrong here

File: src/istgtcontrol.c [hunk 11]
```diff
@@ -600,96 +603,95 @@ exec_refresh(UCTL_Ptr uctl)
...
+
+	/* receive result */
+	rc = uctl_readline(uctl);
+	if (rc != UCTL_CMD_OK) {
+		return (rc);
+	}
+	arg = trim_string(uctl->recvbuf);
+	result = strsepq(&arg, delim);
+	strupr(result);
+		if (strcmp(result, "OK") != 0) {
```
Comment (vishnuitta) [inline src/istgtcontrol.c:693]: indentation gone wrong here

File: src/istgtcontrol.c [hunk 12]
```diff
@@ -1029,14 +1034,14 @@ exec_dump(UCTL_Ptr uctl)
...
-			while((var = strsepq(&arg, delim)) != NULL)
+			while ((var = strsepq(&arg, delim)) != NULL)
 				printf("%s\n", var);
 			flag = 1;
 			continue;
-		}
-		else if(strcmp(type, "CONN") == 0) {
-			if(flag == 1) {
-				printf("c#      ISID            TSIH   CID   T_IP   I_IP            NAME\n");
+		}	else if (strcmp(type, "CONN") == 0) {
```
Comment (vishnuitta) [inline src/istgtcontrol.c:1047]: here also, indentation gone wrong

**Review by vishnuitta — APPROVED** [review]
changes are good

**Comment by rishabhp99** [comment]
Any more changes required ?

**Pull request merged**